neuralmonkey/decoders/autoregressive.py

```diff
@@ -161,10 +161,14 @@ def decoding_w(self) -> tf.Variable:
                 initializer=tf.glorot_uniform_initializer())
 
     @tensor
-    def decoding_b(self) -> tf.Variable:
+    def decoding_b(self) -> Optional[tf.Variable]:
+        if self.tie_embeddings:
```

Comment:
Už jsem to asi někde psal a zapadlo to.. Připadá mi validní případ, kdy děláme tie embeddings a chceme užít biasu. Proto bych sem hodil option kterej to (ale jen při tied embeddinzích a defaultně bude vypnutej) bude kontrolovat.

Reply:
to bych uz do tohohle PR netahal